components/src/core/components/TableSidebar/TableSidebar.vue

```diff
@@ -26,7 +26,7 @@
                   ? 'w-75'
                   : 'w-100'
               "
-              :tooltip="!isLeftPanelOpen ? buttonData.label : null"
+              :tooltip="buttonData.label ? buttonData.label : null"
```

Comment:
Previously, the tooltip is only shown when the left pannel is collapsed. Did someone asked to remove that behavior?

Reply:
yes. In the SOS they said. Nirmal and you also agreed.

Reply:
when the content in the main button in the left panel is too lengthy, (ex: add job vacancy) it will shorten the text and adds three dots and it happens from css. So, mike said we have to add a tooltip when it shows three dots.  We cannot do that right. So, Nirmal and You also agreed to make the tooltip always visible on hovering over it either leftpanel is collapsed or not.